src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -122,8 +122,12 @@ namespace OpenMS
     scores_to_use.setValidStrings("use_nr_peaks_score", ListUtils::create<String>("true,false"));
     scores_to_use.setValue("use_total_xic_score", "true", "Use the total XIC score", ListUtils::create<String>("advanced"));
     scores_to_use.setValidStrings("use_total_xic_score", ListUtils::create<String>("true,false"));
+    scores_to_use.setValue("use_total_mi_score", "true", "Use the total MI score", ListUtils::create<String>("advanced"));
+    scores_to_use.setValidStrings("use_total_mi_score", ListUtils::create<String>("true,false"));
     scores_to_use.setValue("use_sn_score", "true", "Use the SN (signal to noise) score", ListUtils::create<String>("advanced"));
     scores_to_use.setValidStrings("use_sn_score", ListUtils::create<String>("true,false"));
+    scores_to_use.setValue("use_mi_score", "true", "Use the MI (mutual information) score", ListUtils::create<String>("advanced"));
+    scores_to_use.setValidStrings("use_mi_score", ListUtils::create<String>("true,false"));
```

Comment:
do you think it makes sense to turn them on by default? Does it improve our recall?